appIndicator.js

```diff
@@ -315,6 +315,7 @@ class AppIndicatorProxy extends DBusProxy {
                 this.set_cached_property(propertyName, null);
                 this._cancellables.delete(propertyName);
                 delete this._changedProperties[propertyName];
+                console.log(`Error when calling 'Get(${propertyName})' in ${this.gName}, ${this.gObjectPath}, org.freedesktop.DBus.Properties, ${this.gInterfaceName}`);
```

Comment:
Please use `Util.Logger.debug` that can be controlled via `G_MESSAGES_DEBUG`

Reply:
Done.

Reply:
But maybe this should be marked as a Warning... What do you think?

Reply:
I'd agree, I just don't want to confuse people that seems to be watching too much their journals :joy:

Reply:
You are right. Let's keep it as a debug :-D

Reply:
@3v1n0 @sergio-costas Just for the record, [according to the GJS docs](https://gjs.guide/extensions/development/debugging.html) the `console` API is actually recommended for logging, and `console.debug()` can be used to log at `GLib.LogLevelFlags.LEVEL_DEBUG`.

Reply:
@ferdnyc indeed, but the logging part needs to be refactored to use that. Plus I want to use a different logging domain.

The two things are doing basically the same thing, though.